requirements/base-requirements.in

```
@@ -39,7 +39,7 @@ django-transfer==0.4
 django-two-factor-auth~=1.12
 django-user-agents==0.4.0
 django-websocket-redis==0.6.0
-django~=2.2.19
+django==2.2.20
```

This should not be limited to the patch. Otherwise dependabot will not find new versions
```suggestion
django~=2.2.20
```